jupyter_server/terminal/handlers.py

```diff
@@ -11,6 +11,14 @@
 from ..base.zmqhandlers import WebSocketMixin
 
 
+class TerminalHandler(JupyterHandler):
+    """Render the terminal interface."""
+    @web.authenticated
+    def get(self, term_name):
+        self.write(self.render_template('terminal.html',
+                   ws_path="terminals/websocket/%s" % term_name))
+
+
```

Comment:
https://github.com/jupyterlab/nbclassic/blob/0e30d80de5546b50c956ad33ec6e86b6cd554dba/nbclassic/terminal/handlers.py#L14-L19

Reply:
I see.  That's one of the side-effects of the transition from notebook to jupyter_server.  That makes sense.  Thank you.